runtime/gc_stats/MarkVLHGCStats.hpp

```diff
@@ -80,6 +80,9 @@ class MM_MarkVLHGCStats : public MM_MarkVLHGCStatsCore
 	UDATA _splitArraysProcessed; /**< The number of array chunks (not counting parts smaller than the split size) processed by this thread */
 #endif /* J9MODRON_TGC_PARALLEL_STATISTICS */
 
+	U_64 _concurrentGCThreadsStartTimeSum; /**< The sum of all gc cpu thread times when concurrent gc work began */
+	U_64 _concurrentGCThreadsEndTimeSum; /**< The sum of all gc cpu thread times when concurrent gc work ended */
```

Comment:
you could add CPU the names

Reply:
renamed it to `_concurrentGCThreadsCPUStartTimeSum ` as per the latest commit.